pyeasee/__main__.py

```diff
@@ -335,10 +335,14 @@ def str_fixed_length(myStr, length: int):
     return myStr
 
 
+def main():
+    asyncio.run(async_main())
+
+
 if __name__ == "__main__":
     import time
 
     s = time.perf_counter()
-    asyncio.run(main())
+    asyncio.run(async_main())
```

Comment:
Best way is to send in the event loop from the context. I'll share an example tomorrow